packages/types/src/registry.ts

```diff
@@ -20,17 +31,37 @@ import { ViewProtocolService } from '@buf/penumbra-zone_penumbra.connectrpc_es/p
  */
 
 export const typeRegistry = createRegistry(
-  IbcQueryService,
   CustodyProtocolService,
-  SimulationService,
-  TendermintProxyService,
   ViewProtocolService,
+
+  IbcQueryService,
+
+  AppService,
+  ChainService,
+  CompactBlockService,
+  DexService,
+  DexSimulationService,
+  GovernanceService,
+  SctService,
+  ShieldedPoolService,
+  StakeService,
+  TendermintProxyService,
 );
 
 export const serviceTypeNames = [
-  IbcQueryService.typeName,
   CustodyProtocolService.typeName,
-  SimulationService.typeName,
-  TendermintProxyService.typeName,
   ViewProtocolService.typeName,
+
+  IbcQueryService.typeName,
+
+  AppService.typeName,
+  ChainService.typeName,
+  CompactBlockService.typeName,
+  DexService.typeName,
+  DexSimulationService.typeName,
+  GovernanceService.typeName,
+  SctService.typeName,
+  ShieldedPoolService.typeName,
+  StakeService.typeName,
+  TendermintProxyService.typeName,
 ] as const;
```

Comment:
It looks like we can have a single array with  `[CustodyProtocolService, ViewProtocolService...]` and spread them into `createRegistry` and do `serviceTypeNames = arr.map(s => s.typeName)`.

Reply:
That single list could also be imported and used here too: https://github.com/penumbra-zone/web/pull/386/files#diff-b9be68cac5d37d21289e20d8d7fcb88d993de52b16e69a6500255f3fde092ae0R85-R98

Reply:
last time i tried generating this list instead of writing a literal const, i was unable to satisfy ts and keep the injected bundle small.

Reply:
that other list is a list of proxies - they are distinct, and different things happen to them. i will look into using one or the other for deduplication.